.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -55,37 +55,6 @@ _If your PR includes UI changes, please replace this line with details on how
 accessibility is impacted and tested. For more info, check out the
 [Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._
 
-## Added/updated tests?
```

Comment:
@mirie should we remove this section? I feel like this section is important for OS contributors who sometimes need help writing tests? I think its also good to be able to rationalize about the cases where one is not writing tests for the feature.

Reply:
I agree with @Ridhwana here, I think this section is worth keeping.